static/app/gettingStartedDocs/javascript/astro.tsx

```diff
@@ -206,20 +203,74 @@ const onboarding: OnboardingConfig = {
 };
 
 const replayOnboarding: OnboardingConfig = {
-  install: () => getInstallConfig(),
+  install: () => [
+    {
+      ...getInstallConfig()[0],
+      additionalInfo:
+        'Session Replay is enabled by default when you install the Astro SDK!',
```

Comment:
do we need to state that it's enabled by default twice? it's here and also on the first line of the configure step

Reply:
Yeah I was thinking better twice than confuse people why there's the configure section but you're right, let's leave it here and remove it from the configure section.